Dockerfile.konflux

```diff
@@ -58,4 +59,4 @@ LABEL com.redhat.component="odh-dashboard-container" \
       io.openshift.expose-services="" \
       io.k8s.display-name="odh-dashboard" \
       io.k8s.description="odh-dashboard" \
-      com.redhat.license_terms="https://www.redhat.com/licenses/Red_Hat_Standard_EULA_20191108.pdf"
```

Comment:
Why have you removed the newline here?
We always aim to have newlines at the end of files.